models/order/src/order-from-quote-draft/generator.ts

```diff
@@ -0,0 +1,26 @@
+import { Reference } from '@commercetools-test-data/commons';
+import {
+  fake,
+  Generator,
+  oneOf,
+  sequence,
+} from '@commercetools-test-data/core';
+import { orderState, paymentState, shipmentState } from '../constants';
+import { TOrderFromQuoteDraft } from '../types';
+
+// https://docs.commercetools.com/api/projects/orders#orderfromquotedraft
+
+const generator = Generator<TOrderFromQuoteDraft>({
+  fields: {
+    version: sequence(),
+    quote: fake(() => Reference.random().typeId('quote')),
+    quoteStateToAccepted: fake(() => false),
```

Comment:
```suggestion
    quote: Reference.random().typeId('quote'),
    quoteStateToAccepted: false,
```

I don't think you need to wrap directly declared values in `fake` if you arent using `faker` (eg `f.number...` or whatever)

Reply:
strong point, tbh.